script/build-guides.js

```diff
@@ -46,9 +47,17 @@ walk('guides')
     console.log(file);
     guides += `* [${visibleName}](https://aws.github.io/amazon-chime-sdk-js/modules/${webFilename})\n`;
     result += `/**\n${data}\n */\nnamespace ${namespaceName} {}\n\n`;
+    namespaces.push(namespaceName);
   });
 
-fs.writeFileSync('./guides/docs.ts', result, 'utf8');
+let namespaceExports = `\nexport {\n ${namespaces.join(',\n ')}\n}`;
+fs.writeFileSync('./guides/docs.ts', `${result+namespaceExports};`, 'utf8');
+const customModuleNameAnnotation =
+`/**
+ * @packageDocumentation
+ * @module Guides
+ */`;
+fs.writeFileSync('./guides/index.ts', `${customModuleNameAnnotation}\n${namespaceExports} from './docs';`, 'utf8');
```

Comment:
What is the difference between index.ts and doc.ts?

Reply:
To give the "guides/docs.ts" a different custom name like "Guides", I had to export the namespaces that are generated at runtime in the "docs.ts". Once exported I had to re-export them from "index.ts" where I could give a custom module name to that "index.ts". Giving a custom name at the start of the "docs.ts" did not helped as that file complains of not being a module. And I tried various variations but still the name in generated docs remained "guides/docs.ts" and not what I provided. So as Richard suggested to have index.ts and export everything from docs.ts, that helped.